contracts/RiskManagerV1.sol

```diff
@@ -45,6 +47,16 @@ contract RiskManagerV1 is Auctioneer, Ownable {
 
     uint256 public constant DEPOSIT_LIQUIDATION_IN_PROGRESS_STATE = 10;
     uint256 public constant DEPOSIT_LIQUIDATED_STATE = 11;
+    // Auction will not be opened if the deposit collateralization level does not
+    // fall below this collateralization threshold.
+    // Risk manager should open a coverage pool auction for only those deposits
+    // that nobody else is willing to purchase. The default value can be updated
+    // by the governance in two steps. First step is to begin the update process
+    // with the new value and the second step is to finalize it after
+    // GOVERNANCE_TIME_DELAY has passed.
+    uint256 public collateralizationThreshold = 101; // percent
+    uint256 public newCollateralizationThreshold;
+    uint256 public collateralizationThresholdInitiated;
```

Comment:
We have `auctionLengthChangeInitiated` so this one should be `collateralizationThresholdChangeInitiated`.

Reply:
81c3fe3